backends-velox/src/main/scala/org/apache/spark/sql/execution/datasources/velox/VeloxFormatWriterInjects.scala

```diff
@@ -47,6 +47,15 @@ trait VeloxFormatWriterInjects extends GlutenFormatWriterInjectsBase {
       nativeConf: java.util.Map[String, String]): OutputWriter = {
     val originPath = path
 
+    // Created the hdfs path if not existed.
+    if (path.substring(0, 5).equals("hdfs:")) {
```

Comment:
Maybe declare `hdfs:` as a const variable, and use its length in substring.

Reply:
@rui-mo Updated.

Reply:
Nit:
`path.startsWith(hdfsScheme)`

BTW, not related to this PR: `originPath` looks redundant in this part of code. Can we just use `path` instead?`?